config/owasp/suppressions.xml

```diff
@@ -2,44 +2,38 @@
 <suppressions
   xmlns="https://jeremylong.github.io/DependencyCheck/dependency-suppression.1.3.xsd">
 
-	<suppress>
-    <notes>
-			Declared as False positive on library com.nimbusds:lang-tag #3594
-			https://github.com/jeremylong/DependencyCheck/issues/3594
-			CVE-2020-23171 Reference https://tools.hmcts.net/jira/browse/CCD-1869
-    </notes>
-    <packageUrl regex="true">^pkg:maven/com\.nimbusds/lang\-tag@.*$</packageUrl>
-    <cpe>cpe:/a:nim-lang:nim-lang</cpe>
-    <cve>CVE-2020-23171</cve>
-	</suppress>
-
-  <suppress>
-    <notes><![CDATA[
-     It's a false positive - spring-security version is 5.3.x (see: https://pivotal.io/security/cve-2018-1258)
-    ]]></notes>
-    <cve>CVE-2018-1258</cve>
-  </suppress>
-
+  <!--Please add all the false positives under the below section-->
   <suppress>
-    <notes>These CVE's are coming from Dhowden tag library and it impacts only MP3/MP4/OGG/FLAC metadata parsing
-      library. Also it is declared as false positive in https://github.com/jeremylong/DependencyCheck/issues/3043
+    <notes>False Positives
+      CVE-2018-1258 refer https://tools.hmcts.net/jira/browse/CCD-645
+      CVE-2020-29242 refer https://tools.hmcts.net/jira/browse/CCD-48
+      CVE-2020-29243 refer https://tools.hmcts.net/jira/browse/CCD-126
+      CVE-2020-29244 refer https://tools.hmcts.net/jira/browse/CCD-338
+      CVE-2020-29245 refer https://tools.hmcts.net/jira/browse/CCD-191
+      CVE-2007-1651 refer https://tools.hmcts.net/jira/browse/CCD-39
+      CVE-2007-1652 refer https://tools.hmcts.net/jira/browse/CCD-313
+      CVE-2016-1000027 refer https://tools.hmcts.net/jira/browse/CCD-3246
     </notes>
+    <cve>CVE-2018-1258</cve>
```

Comment:
shall we remove CVE-2018-1258 entirely?

Reply:
suppression is required for this

Reply:
I thought this could be removed as like https://github.com/hmcts/ccd-case-disposer/pull/74/files